### DISCUSSION TITLE
Run only the ancestry PCA projection - skip scoring?

### DISCUSSION THREAD

**Discussion by csjohnson23** [PGScatalog/pgsc_calc#372] [Q&A] [answered]
I'm messing with filters to better align my target dataset with the ancestry panel. To do this, I have to run the pipeline repeatedly with different MAF and missingness values.  The actual calculated PGS are irrelevant to this process but they take forever to run and generate the score report (I'm on a crowded cluster) - is there a way to only run the ancestry calculation/projection piece of the pipeline, and skip the scoring? Alternatively, is there a point at which I can safely interrupt the pipeline execution and look at the ancestry pca projection before all the scoring is done? Thanks!

**Comment by nebfield** [comment, marked as answer]
`--only_projection` might be helpful 😅

https://pgsc-calc.readthedocs.io/en/latest/reference/params.html#developer-options has some other debug parameters to control subworkflow execution
